contracts/base-factory/src/msg.rs

```diff
@@ -9,19 +9,16 @@ pub struct InstantiateMsg {
     pub params: BaseMinterParams,
 }
 
-pub type Extension = Option<Empty>;
-
 pub type BaseMinterCreateMsg = CreateMinterMsg<Extension>;
 
 pub type ExecuteMsg = Sg2ExecuteMsg<Extension>;
 
 pub type BaseUpdateParamsMsg = UpdateMinterParamsMsg<Extension>;
 
 #[cw_serde]
-pub enum SudoMsg<T> {
-    UpdateParams(Box<T>),
+pub enum SudoMsg {
```

Comment:
Don't think we should get rid of the generic `SudoMsg<T>` because future minters can use it to update different params.